client/ayon_maya/plugins/load/load_image_plane.py

```diff
@@ -85,7 +85,8 @@ class ImagePlaneLoader(plugin.Loader):
 
     product_types = {"image", "plate", "render"}
     label = "Load imagePlane"
-    representations = {"mov", "exr", "preview", "png", "jpg", "jpeg"}
+    representations = {"*"}
+    extensions = {"mov", "mp4", "exr", "preview", "png", "jpg", "jpeg"}
```

Comment:
Silly me, "preview" is not an extension..... or is it!? 
```suggestion
    extensions = {"mov", "mp4", "exr", "png", "jpg", "jpeg"}
```